lib/publiccloud/utils.pm

```diff
@@ -134,6 +134,7 @@ sub register_addon {
         ssh_add_suseconnect_product($remote, get_addon_fullname($addon), program => $program, version => '${VERSION_ID}', arch => $arch, params => "-r " . get_required_var('SCC_REGCODE_LTSS'), timeout => $timeout, retries => $retries, delay => $delay);
     } elsif (is_ondemand) {
         record_info($addon, 'This is on demand image, we will not register this addon.');
+	return;
```

Comment:
correct me if I wrong , it is just to avoid record_info at the end ? ( totally fine by me just checking myself  that I got it right ... )

Reply:
Yes :smile: